CHANGELOG.md

```diff
@@ -1,217 +1,241 @@
-# Release 8.0.0
+# Release 9.0.0
 
 **NOTE:**
-_This changelog only contains release notes for PWA Studio 8.0.0._
+_This changelog only contains release notes for PWA Studio 9.0.0._
```

Comment:
We should start referring to the product as `PWA Studio & Venia` in all new documentation and when refactoring any existing documentation.